<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T12306</class>
 <widget class="QDialog" name="T12306">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T12306</string>
  </property>
  <widget class="QComboBox" name="qidian_comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>北京</string>
   </property>
  </widget>
  <widget class="QComboBox" name="zhongdian_comboBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>龙岩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chaxun_pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>5</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-M-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>391</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>11</number>
   </property>
   <property name="columnCount">
    <number>14</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>57</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
